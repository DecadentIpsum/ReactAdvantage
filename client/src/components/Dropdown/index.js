import React, { Component } from 'react';
import { Dropdown as PrimeDropdown } from 'primereact/components/dropdown/Dropdown';

import './index.css';

export default class Dropdown extends Component {
    render() {
        return (
            <PrimeDropdown className="dropdown" {...this.props} />
        );
    }
}
