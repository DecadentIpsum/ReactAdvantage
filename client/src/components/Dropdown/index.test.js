import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import Dropdown from './index';

it('renders correctly', () => {
    const component = shallow(<Dropdown />);

    expect(component).toMatchSnapshot();
});
