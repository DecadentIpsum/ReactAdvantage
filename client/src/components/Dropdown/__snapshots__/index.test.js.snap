// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Dropdown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "appendTo": null,
      "autoFocus": false,
      "autoWidth": true,
      "className": null,
      "dataKey": null,
      "disabled": false,
      "editable": false,
      "filter": false,
      "filterPlaceholder": null,
      "id": null,
      "inputId": null,
      "itemTemplate": null,
      "onChange": null,
      "onContextMenu": null,
      "onMouseDown": null,
      "optionLabel": null,
      "options": null,
      "panelClassName": null,
      "panelStyle": null,
      "placeholder": null,
      "required": false,
      "scrollHeight": "200px",
      "showClear": false,
      "style": null,
      "tabIndex": null,
      "value": null,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "appendTo": null,
        "autoFocus": false,
        "autoWidth": true,
        "className": null,
        "dataKey": null,
        "disabled": false,
        "editable": false,
        "filter": false,
        "filterPlaceholder": null,
        "id": null,
        "inputId": null,
        "itemTemplate": null,
        "onChange": null,
        "onContextMenu": null,
        "onMouseDown": null,
        "optionLabel": null,
        "options": null,
        "panelClassName": null,
        "panelStyle": null,
        "placeholder": null,
        "required": false,
        "scrollHeight": "200px",
        "showClear": false,
        "style": null,
        "tabIndex": null,
        "value": null,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
